rules/windows/file/file_event/file_event_win_powershell_exploit_scripts.yml

```diff
@@ -26,9 +26,10 @@ references:
     - https://github.com/Kevin-Robertson/Powermad
     - https://github.com/adrecon/ADRecon
     - https://github.com/adrecon/AzureADRecon
+    - https://github.com/sadshade/veeam-creds/blob/main/Veeam-Get-Creds.ps1
```

Comment:
As per sigma convention all github links have to be permalinks. Please apply for all.